### PR TITLE
NCL-1885 Add available versions information in product import process

### PR DESCRIPTION
**Pull request by matedo1**

### CODE REVIEW

File: ui/app/styles/directives.css [hunk 1]
```diff
@@ -108,3 +108,16 @@
     margin: 2px 0;
     border-radius: 1px;
 }
+
+
+/**
+ * product-import-bc-form
+ */
+.available-versions {
```
Comment (alexcreasy) [inline ui/app/styles/directives.css]: Can you fix the indentation so it matches the rest of the file please?

**Comment by alexcreasy** [comment]
@matedo1 Looks good to me, if you could just fix the indentation on directives.css. Thanks.

**Comment by matedo1** [comment]
@alexcreasy Thanks for quick review! I pushed new commit with formatting. There were more places with different indentation so I reformatted whole file. I will merge it once tests are done.

**Comment by alexcreasy** [comment]
@matedo1 great, thanks for fixing the file, the diff I saw was only a snippet.

**Pull request merged**